js/start.js

```diff
@@ -274,6 +277,39 @@ function onboard() {
   return onboardDeferred.promise();
 }
 
+function fetchVerifiedMods() {
```

Comment:
This function is run on a loop to update the list of verified moderators once an hour. I'm open to suggestions if there's a better way to do the loop, and/or to incorporate the verified moderator fetch into the startup sequence below.